### ISSUE TITLE
user.authentication.success has vhost set to 'none' and is misleading

### ISSUE THREAD

**Issue by mstefaniuk** [rabbitmq/rabbitmq-event-exchange#13]
I have simple test case in C# - user connects to virtual host called `vhost`:
```c#
    var factory = new ConnectionFactory
    {
        UserName = "username",
        Password = "password",
        VirtualHost = "vhost"
    };

    using (var connection = factory.CreateConnection())
    {
        Check.That(connection).IsNotNull();
    }
```

It works okay but event published has property `vhost` with value `none`:
```
Exchange 	amq.rabbitmq.event
Routing Key 	user.authentication.success
Properties 	
  timestamp:	1459239111949
  delivery_mode:	2
  headers:	
    connection_type:	network
    name:	username
    host:	{0,0,0,0,0,0,0,1}
    vhost:	none
    connection_name:	[::1]:54155 -> [::1]:5672
    peer_host:	{0,0,0,0,0,0,0,1}
    peer_port:	54155
    protocol:	{0,9,1}
    auth_mechanism:	PLAIN
    ssl:	false
```

but it should reflect host that user is connected to. It has been checked on RabbitMQ 3.6.0.

**Comment by michaelklishin** [comment]
Authentication is actually a two step process: users first authenticate, then open a connection to a vhost. We don't have any vhost information at the time the former happens. See the `connection_created` event.

**Comment by michaelklishin** [comment]
We could emit both events at the same time but I'm not convinced it necessary and would require internal changes that are unnecessary otherwise.

**Comment by mstefaniuk** [comment]
@michaelklishin I would suggest to remove `vhost` property from this event because it is misleading.

**Comment by michaelklishin** [comment]
@mstefaniuk I agree.

**Comment by hairyhum** [comment]
Fixed in rabbitmq/rabbitmq-common#79
